### PR TITLE
Ported hammer based secnario to Nailgun capsule-sync

### PR DESCRIPTION
**Pull request by vijay8451**
Below Scenario moved to nailgun from hammer:

 2. test_capsule.py
    - Scenario_capsule_sync

****Note**: Testing is pending for these above.**

### CODE REVIEW

File: tests/upgrades/test_capsule.py [hunk 1]
```diff
@@ -81,42 +83,27 @@ def test_pre_user_scenario_capsule_sync(self):
         :expectedresults: The repo/rpm should be synced to satellite
 
          """
-        _, env_name = hammer.hammer_determine_cv_and_env_from_ak(
-            self.activation_key, '1')
+        ak = entities.ActivationKey(organization=self.org_id).search(
+            query={'search': 'name={}'.format(self.activation_key)})[0]
+        env_id = ak.environment.id
+        env_name = ak.environment.read().name
```
Comment (jyejare) [inline tests/upgrades/test_capsule.py]:
```suggestion
         ak_env = ak.environment.read()
```
Then, You can ```ak_env``` as:
```ak_env.id``` and ```ak_env.name```

File: tests/upgrades/test_capsule.py [hunk 2]
```diff
@@ -81,42 +83,27 @@ def test_pre_user_scenario_capsule_sync(self):
...
+        prod = entities.Product(
+            name=self.prod_name, organization=self.org_id).create()
+        self.assertEqual(self.prod_name, prod.name)
```
Comment (jyejare) [inline tests/upgrades/test_capsule.py]: Lets remove this small and setup asserts.

File: tests/upgrades/test_capsule.py [hunk 3]
```diff
@@ -81,42 +83,27 @@ def test_pre_user_scenario_capsule_sync(self):
...
-            env_data,
-            env_name,
-            'id'
-        )
-        print(hammer.hammer_content_view_promote_version(
-            self.cv_name, cv_ver, env_id, self.org_id))
+        repo = entities.Repository(
+            product=self.prod_name, name=self.repo_name,
+            url=self.repo_url).create()
+        self.assertEqual(self.repo_name, repo.name)
```
Comment (jyejare) [inline tests/upgrades/test_capsule.py]: same as above

**Comment by san7ket** [comment]
Code looks good , test

**Comment by vijay8451** [comment]
**- Testing completed with some minor changes and new changed are pushed too:**

++++++++++++++++++++++++++++++++++++++++++++++++++++++
robottelo]# pytest -v tests/upgrades/test_capsule.py::Scenario_capsule_sync -m pre_upgrade
============================================================================ test session starts ============================================================================
platform linux2 -- Python 2.7.5, pytest-3.6.1, py-1.7.0, pluggy-0.6.0 -- /home/vijsingh/my_projects/py2test/bin/python2
cachedir: .pytest_cache
rootdir: /home/vijsingh/my_projects/ROBO/6.4.z/robottelo, inifile:
plugins: services-1.3.0, mock-1.10.0
2019-01-08 13:02:22 - robottelo - INFO - Started setUpClass: robottelo.test/TestCase
collected 2 items / 1 deselected                                                                                                                                            

tests/upgrades/test_capsule.py::Scenario_capsule_sync::test_pre_user_scenario_capsule_sync 

PASSED2019-01-08 13:04:42 - robottelo - INFO - Started tearDownClass: tests.upgrades.test_capsule/Scenario_capsule_sync


================================================================= 1 passed, 1 deselected in 140.71 seconds ==================================================================

robottelo]# pytest -v tests/upgrades/test_capsule.py::Scenario_capsule_sync -m post_upgrade
============================================================================ test session starts ============================================================================
platform linux2 -- Python 2.7.5, pytest-3.6.1, py-1.7.0, pluggy-0.6.0 -- /home/vijsingh/my_projects/py2test/bin/python2
cachedir: .pytest_cache
rootdir: /home/vijsingh/my_projects/ROBO/6.4.z/robottelo, inifile:
plugins: services-1.3.0, mock-1.10.0
2019-01-08 13:12:35 - robottelo - INFO - Started setUpClass: robottelo.test/TestCase
collected 2 items / 1 deselected                                                                                                                                            

tests/upgrades/test_capsule.py::Scenario_capsule_sync::test_post_user_scenario_capsule_sync 

PASSED2019-01-08 13:12:42 - robottelo - INFO - Started tearDownClass: tests.upgrades.test_capsule/Scenario_capsule_sync


================================================================== 1 passed, 1 deselected in 6.44 seconds ===================================================================

robottelo]# pytest -v tests/upgrades/test_capsule.py::Scenario_capsule_sync_2 -m post_upgrade
============================================================================ test session starts ============================================================================
platform linux2 -- Python 2.7.5, pytest-3.6.1, py-1.7.0, pluggy-0.6.0 -- /home/vijsingh/my_projects/py2test/bin/python2
cachedir: .pytest_cache
rootdir: /home/vijsingh/my_projects/ROBO/6.4.z/robottelo, inifile:
plugins: services-1.3.0, mock-1.10.0
2019-01-08 13:13:55 - robottelo - INFO - Started setUpClass: robottelo.test/TestCase
collected 1 item                                                                                                                                                            

tests/upgrades/test_capsule.py::Scenario_capsule_sync_2::test_post_user_scenario_capsule_sync_2

PASSED2019-01-08 13:16:51 - robottelo - INFO - Started tearDownClass: tests.upgrades.test_capsule/Scenario_capsule_sync_2


======================================================================== 1 passed in 176.26 seconds =========================================================================

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

**Pull request merged**